.config/requirements.in

```diff
@@ -1,3 +1,3 @@
 importlib-resources ; python_version < '3.10'
-Jinja2==3.1.2
+Jinja2>=3.1.2
```

Comment:
@ssbarnea why do we not want to bump this to `>=3.1.3`? This would also help with the dependabot security alert? Is it because we want dependency bumps to be part of next major release _only_.

Reply:
That is because it create conflicts with other packages that pin it down, breaking the builds. Security is not a good enough reason to bump minimal version.

Another issue is that being aggressive with these break building of system packages (rpms, debs). Sometimes it can take weeks or even months for the packagers to upgrade the dependencies.